content/docs/user-guide/programming/_index.md

```diff
@@ -43,6 +43,20 @@ Examples of high-level products that use these frameworks are:
 * **GUI tools**: Tools with a graphical user experience are used for developing O3DE projects, such as **O3DE Editor**, **Asset Processor**, and **Material Editor**.
 
 
+### O3DE directories
+
+If you're interested in developing O3DE, creating components and Gems, or programming in your project, it's helpful to be familiar with the subdirectories in O3DE's source. 
```

Comment:
You don't cover all the directories here. Is it fair to say "the following", implying a list of key directories?
```suggestion
If you're interested in developing O3DE, creating components and Gems, or programming in your project, it's helpful to be familiar with the following subdirectories in O3DE's source:
```